sdk/src/main/java/com/mapbox/maps/MapInitOptions.kt

```diff
@@ -16,6 +16,7 @@ import android.util.AttributeSet
  * @property mapOptions Describes the map options value when using a MapView.
  * @property initialCameraOptions The Initial Camera options when creating a MapView.
  * @property textureView Flag indicating to use a TextureView as render surface for the MapView. Default is false.
+ * @property styleUri The styleUri will applied for the MapView in the onStart lifecycle event if no style is set. Default is [Style.MAPBOX_STREETS].
```

Comment:
we should also document that if the field is set to null, no style will be loaded(matches the old behaviour).